components/imageCard.js

```diff
@@ -50,12 +58,80 @@ export default function ImageCard() {
       ) : (
         ""
       )}
-    </>
+
+      <ButtonContainer>
+        <DeleteButton>
+          <Image
+            className="inactiveicon"
+            src={deleteIcon}
+            alt="Delete Icon"
+            height={100}
+            width={100}
+          />
+          <Image
+            className="activeicon"
+            src={deleteIconFilled}
+            alt="Delete Icon Filled"
+            height={100}
+            width={100}
+          />
+        </DeleteButton>
+
+        <LikeButton onClick={() => addFavorite(displayImage)}>
+          <Image src={likeIcon} alt="Like Icon" height={80} width={80} />
+          <Image
+            src={likeIconFilled}
+            alt="Like Icon Filled"
+            height={80}
+            width={80}
+          />
+        </LikeButton>
+      </ButtonContainer>
+    </div>
   );
 }
 
 const ImageContainer = styled.div`
-  position: relative;
+  display: flex;
   width: 100%;
   height: 400px;
 `;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  padding-top: 60px;
+`;
+
+const DeleteButton = styled.button`
+  border: none;
+  background: none;
+
+  .activeicon {
+    display: none;
+  }
+  /*
+  :hover .inactiveicon {
+    visibility: hidden;
+  }
+  .activeicon {
+    visibility: visible;
+  } */
+`;
+
+const LikeButton = styled.button`
+  border: none;
+  background: none;
+`;
+
```

Comment:
Try to create one `Button` component that you can use for all of your buttons (use props for different variants)

Reply:
I will get deeper into this. I was adopting the functions to what I want eventually (fetching data on button click instead of on reloading the page as it is now).